<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="6" hostname="SUMIT-SONI" name="test.TestVeriday" tests="8" failures="0" timestamp="30 Nov 2020 15:23:14 GMT" time="61.822" errors="1">
  <testcase name="loginTest" time="14.584" classname="test.TestVeriday"/>
  <testcase name="CreateNewFormTest" time="47.238" classname="test.TestVeriday">
    <error type="java.lang.IllegalArgumentException" message="Keys to send should be a not null CharSequence">
      <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at BasePage.BasePage.enterTextwithoutclear(BasePage.java:483)
at pages.DigitalWorkspacePage.setEmailRequire(DigitalWorkspacePage.java:142)
at pages.DigitalWorkspacePage.CreateNewForm(DigitalWorkspacePage.java:246)
at test.TestVeriday.CreateNewFormTest(TestVeriday.java:123)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- CreateNewFormTest -->
  <testcase name="CreateTeamUserTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- CreateTeamUserTest -->
  <testcase name="CreateNewWebPageTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- CreateNewWebPageTest -->
  <testcase name="CreatePrimaryUserTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- CreatePrimaryUserTest -->
  <testcase name="ImpersonateUserTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- ImpersonateUserTest -->
  <testcase name="SearchUserTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- SearchUserTest -->
  <testcase name="CreateNewEventTest" classname="test.TestVeriday">
    <skipped/>
  </testcase> <!-- CreateNewEventTest -->
</testsuite> <!-- test.TestVeriday -->
